budget-squirrel/src/pages/Login/Login.js

```diff
@@ -17,21 +17,39 @@ export default function Login() {
     const usernameRef = useRef();
     const passwordRef = useRef();
 
+    const [usernameError, setUsernameError] = useState(false);
```

Comment:
Do you think we should look int to base front end validation? Like the Username / Password field can't be empty. or the username must match email passwords? I think it could reduce traffic to our back-end.

Reply:
Yeah that probably would be nice to have